src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -676,43 +680,43 @@ Metric retention periods are not editable.
   
 </CollapserGroup>
 
-## Edit retention values [#adjust-retention]
+## Edit data retention [#adjust-retention]
 
-If you meet [the requirements](#requirements), you can increase the retention for the data types listed in the **Data retention** UI. To do that: on that page, click **Adjust retention**. 
+To view and edit your retention, go to [the data management hub](https://one.newrelic.com/data-management-hub), click **Data retention**, and click **Edit data retention**. There you'll see options for editing data retention or upgrading to Data Plus. If you don't see that, review [the requirements](#requirements) and talk to your New Relic account representative. 
 
-Important things to note about increasing retention: 
+Options for increasing retention: 
 
-* Retention is increased across the board, for all data types listed in the **Data retention** UI. It can't be increased for specific data types. 
-* For each additional 30 days of retention, the cost is $0.05 per GB ingested per month. 
-* Once a higher retention period is set, reducing retention for specific data sources has no impact on billing.
+* If you haven't already, switch to [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), which offers significantly longer data retention and other benefits.
+* For Data Plus or original data option: add data retention. 30 days of additional data retention for your organization costs $0.05 per GB ingested per 30 days.
 
-### Other notes on reducing retention [#reduce-retention]
+Important points about increasing retention: 
 
-When you reduce the retention period for a specific data source, existing data of that type won't immediately disappear. That data might be visible and returned in queries for up to several weeks. We have safeguards in place to ensure you don't accidentally delete data, but this also means that reducing retention values isn't a good strategy for quickly [removing data](#data-deletion). 
+* Retention is increased across the board, for all data types listed in the **Data retention** UI. It cannot be increased only for specific data types. 
```

Comment:
Should we add those data types here or is there another place where they are referenced?